package.json

```diff
@@ -29,31 +29,43 @@
     "Google Cloud Key Management Service (KMS) API"
   ],
   "scripts": {
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "test": "c8 mocha",
-    "lint": "eslint '**/*.js'",
+    "test": "c8 mocha build/test",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "system-test": "mocha build/system-test",
+    "lint": "gts fix && cd samples/ && npm install && cd .. && eslint --fix samples/*.js",
+    "fix": "gts fix",
     "docs": "jsdoc -c .jsdoc.js",
-    "system-test": "mocha system-test/ smoke-test/ --timeout 600000",
-    "fix": "eslint '**/*.js' --fix",
     "docs-test": "linkinator docs",
     "predocs-test": "npm run docs",
-    "prelint": "cd samples; npm link ../; npm i"
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r protos build/",
+    "compile-protos": "compileProtos src",
+    "prepare": "npm run compile"
   },
   "dependencies": {
-    "google-gax": "^1.7.5"
+    "google-gax": "^1.9.0"
   },
   "devDependencies": {
-    "codecov": "^3.0.4",
+    "@types/mocha": "^5.2.5",
+    "@types/node": "^12.0.0",
+    "c8": "^6.0.0",
     "eslint": "^6.0.0",
     "eslint-config-prettier": "^6.0.0",
     "eslint-plugin-node": "^11.0.0",
     "eslint-plugin-prettier": "^3.0.0",
+    "gts": "^1.0.0",
     "jsdoc": "^3.6.2",
     "jsdoc-fresh": "^1.0.1",
     "jsdoc-region-tag": "^1.0.2",
-    "linkinator": "^2.0.0",
-    "mocha": "^7.0.0",
-    "c8": "^7.0.0",
-    "prettier": "^1.13.7"
+    "linkinator": "^1.5.0",
+    "mocha": "^6.1.4",
```

Comment:
A lot of these are out of date.  Could I trouble you to `npx npm-upgrade` and bump all these?

Reply:
cool, updated it, thank you for catching this.